src/auth/auth.controller.spec.ts

```diff
@@ -74,7 +78,67 @@ describe('AuthController', () => {
       });
```

Comment:
nit, would be nice to add the same unit test for jwt token

Reply:
Added